src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -765,22 +757,23 @@ public InputStream retrieveMetadata(String pid, String formatId) throws Exceptio
     /**
      * Overload method for retrieveMetadata with default metadata namespace
      */
-    public InputStream retrieveMetadata(String pid) throws Exception {
```

Comment:
Can this method call retrieveMetadata(pid, formatid)? So we can get rid of the duplicated code.

Reply:
I have updated both overload methods, as well as cleaned up the logging statements. Thank you!